MH-13067, Configuration panel does not work for default workflow

lkiesow opened the pull request:
If a default workflow is configured in `custopm.properties` and hence
automatically selected in the “New Event” wizard, all user input from
the configuration panel is discarded and the default settings are used
instead.

This can be easily tested by configuring the default workflow:

    org.opencastproject.workflow.default.definition=schedule-and-upload

…and selecting the first two options in the configuration panel of the
“New Event” wizard. The summary will already show these values to be set
to false (despite of them being selected earlier).

This patch ensures the configuration panel is looked up when updating
the configuration to ensure the used configuration is always up-to-date.

lkiesow committed:
MH-13067, Configuration panel does not work for default workflow

If a default workflow is configured in `custopm.properties` and hence
automatically selected in the “New Event” wizard, all user input from
the configuration panel is discarded and the default settings are used
instead.

This can be easily tested by configuring the default workflow:

    org.opencastproject.workflow.default.definition=schedule-and-upload

…and selecting the first two options in the configuration panel of the
“New Event” wizard. The summary will already show these values to be set
to false (despite of them being selected earlier).

This patch ensures the configuration panel is looked up when updating
the configuration to ensure the used configuration is always up-to-date.

staubesv reviewed (changes requested):
Hi @lkiesow 

I think this PR makes line 38 and 90 obsolete. Can you please check and remove those lines?

staubesv commented:
Please also note that the configuration panel has another bug: If you select "onhold", the workflow will still publish. But that is another issue...

lkiesow commented:
@staubesv, updated the pull request. had to slightly modify the line I changed before as well.

The workflow definition (onhold issue) should be completely unrelated to this (workflow issue vs admin interface issue).
